client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.js

```diff
@@ -73,12 +75,17 @@ const DataFilesToolbar = ({ scheme }) => {
     });
   };
 
-  const canRename = selectedFiles.length === 1 && scheme === 'private';
-  const canMove = selectedFiles.length > 0 && scheme === 'private';
+  const canRename =
+    selectedFiles.length === 1 && scheme === 'private' && api !== 'googledrive';
+  const canMove =
+    selectedFiles.length > 0 && scheme === 'private' && api !== 'googledrive';
   const canCopy = selectedFiles.length > 0 && scheme === 'private';
   const canDownload =
-    selectedFiles.length === 1 && selectedFiles[0].format !== 'folder';
-  const canTrash = selectedFiles.length > 0 && scheme === 'private';
+    selectedFiles.length === 1 &&
+    selectedFiles[0].format !== 'folder' &&
+    api !== 'googledrive';
```

Comment:
Why no downloading again? Would downloading require a change in file permissions?

Reply:
Yeah, otherwise we'd have to download the file on the server side and pass it on to the client. I tried this in designsafe and it was really messy.